isaric/schemas.py

```diff
@@ -136,7 +136,6 @@ class Visit(BaseModel):
     treatment_intravenous_fluids: bool | None = None
     treatment_nsaid: bool | None = None
     treatment_neuraminidase: bool | None = None
-    treatment_oxygen_interface: bool | None = None
```

Comment:
I am of the opinion that we keep this in the schemas .. this follows my discussion with Rosario today. There might be other ISARIC datasets where there will be matching fields / columns for 'oxygen_interface' ...

Reply:
Good point - I'll add the two oxygen treatment options back into the schema